compiler/frontend/src/org/jetbrains/kotlin/resolve/lazy/descriptors/AbstractLazyMemberScope.kt

```diff
@@ -188,6 +188,12 @@ protected constructor(
 
     protected abstract fun getNonDeclaredProperties(name: Name, result: MutableSet<PropertyDescriptor>)
 
+    override fun definitelyDoesNotContainName(name: Name): Boolean =
+        !getVariableNames().contains(name) &&
+                !getFunctionNames().contains(name) &&
+                getClassifierNames()?.contains(name) != true &&
```

Comment:
Sorry, it's not obvious but this is incorrect semantic usage of the `getClassifierNames()`: it returns null when the scope can't compute classifier names' set efficiently.
And this happens to `LazyClassMemberScope` unfortunately.
It's necessary to implement it properly there to make `definitelyDoesNotContainName` work correctly

Reply:
@dzharkov Fixed this method, please review new implementation